### PR TITLE
Add friday blog post

### PR DESCRIPTION
**Pull request by davecheney**
Blog post with friday details.

Still TODO before landing, get coreos details @kelseyhightower and google details @francesc

### CODE REVIEW

**Comment by davecheney** [comment]
@spf13 can you copy check the docker room section, thanks.

**Comment by davecheney** [comment]
@bketelsen @erikstmartin I believe this post is ready to publish, today if possible.

Google timed out, I'll circle back with them for a followup post at the end of the week.

**Comment by francesc** [comment]
@francesc on github is not the Google's Francesc you're looking for ;)

**Comment by davecheney** [comment]
oh poop.

**Comment by spf13** [comment]
Docker copy LGTM.

File: content/gophercon-2015/gophercon-friday-hack-day.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+linktitle = "GopherCon Friday Hack Day details"
+title = "GopherCon Friday Hack Day details"
++++
+With less than a month til GopherCon 2015 we’ve been hard at work on plans for the Friday 10th Hack Day. The previous year’s unofficial Friday Hack Day was so popular we decided this year to integrate it into the conference proper. 
+
+We're providing the space and the wifi, you bring your favorite project and work together with friends you only get to see in person once a year at GopherCon.
+
+After the brisk start of the two previous days, Friday will arrive at a more leisurely pace with most activities starting at 10 am.
+
+Kicking off in the Mile High Ballroom theatre we will feature a presentation “Investment 101 for Startups” by Robin Vasan, managing director a Mayfield, one of the most prestigious investment firms in the country. Robin will be joined for a question and answer session by the CEO's from two of Mayfield's investments. Paul Dix from InfluxDB and Mitchell Hashimoto from HashiCorp who will share their experiences as founders going through the investment process.
```
Comment (erikstmartin) [inline content/gophercon-2015/gophercon-friday-hack-day.md]: managing director at Mayfield

Comment (davecheney) [inline content/gophercon-2015/gophercon-friday-hack-day.md]: Fixed. I have no idea how I can make so many errors when I just copied and pasted the text from the website.

File: content/gophercon-2015/gophercon-friday-hack-day.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+After Robin’s presentation we’re turning the theatre over to you. This is your chance to step on the main stage and tell the world something about Go.
+
+This year we have three lightning talk categories, Hack and Tell, Call to Action, and Go Tips and Tricks. Registrations for lightning talks will close on the 20th of June so head over to the [registration page and sign up](https://gophercon.typeform.com/to/c6GOUe).
+
+We will post the list of lightning talks on the GopherCon website by the end of June.
+
+Next up is the general area directly adjacent to the theatre, which will be open all day. This is your space to work on anything Go related that takes your fancy. Don’t forget to tweet your creations and tag them with [#gophercon](https://twitter.com/hashtag/gophercon?src=hash).
+
+![Denver Conference Center](/postimages/gophercon-2015/gophercon-friday.pkg)
```
Comment (erikstmartin) [inline content/gophercon-2015/gophercon-friday-hack-day.md]: Colorado Convention Center

Comment (davecheney) [inline content/gophercon-2015/gophercon-friday-hack-day.md]: Fixed

**Comment by nathany** [comment]
Fransesc is @campoy

**Comment by campoy** [comment]
I think I'm late to the party ... sorry!

**Comment by davecheney** [comment]
If you want to add anything send me an email and I'll get the post updated.
We'll also be transposing this information onto the conference website this
week.

**Pull request merged**